src/coreclr/src/jit/compiler.h

```diff
@@ -1130,6 +1130,17 @@ enum class PhaseChecks
     CHECK_ALL
 };
 
+// Specify compiler data that a phase might modify
+enum class PhaseStatus : unsigned
+{
+    PS_MODIFIED_NOTHING    = 0x0,
```

Comment:
Nit: enum class should not need a class prefix, because they can't be used without className.

Reply:
Good point, it is annoying to repeat this.